src/main/kotlin/no/nav/syfo/lagrevedtak/db/DbQueries.kt

```diff
@@ -0,0 +1,114 @@
+package no.nav.syfo.lagrevedtak.db
+
+import java.sql.Connection
+import java.sql.Timestamp
+import java.time.LocalDate
+import no.nav.syfo.application.db.DatabaseInterface
+import no.nav.syfo.lagrevedtak.UtbetaltEvent
+import no.nav.syfo.objectMapper
+import org.postgresql.util.PGobject
+
+fun DatabaseInterface.lagreUtbetaltEvent(
+    utbetaltEvent: UtbetaltEvent
+) {
+    connection.use { connection ->
+        connection.lagreUtbetaltEvent(utbetaltEvent)
+        connection.commit()
+    }
+}
+
+fun DatabaseInterface.lagreUtbetaltEventOgPlanlagtMelding(
+    utbetaltEvent: UtbetaltEvent,
+    planlagteMeldinger: List<PlanlagtMeldingDbModel>
+) {
+    connection.use { connection ->
+        connection.lagreUtbetaltEvent(utbetaltEvent)
```

Comment:
Åpnes det en ny connection her i connection.lagreUtbetaltEvent? og det burde bare gjøres en commit for hele greia?

Reply:
Vi ble enige om at det var riktig sånn :)